kurtosis-module/impl/participant_network/participant_network.go

```diff
@@ -261,6 +262,22 @@ func LaunchParticipantNetwork(
 		// Each CL node will be paired with exactly one EL node
 		elClientCtx := allElClientContexts[idx]
 
+		var mevBoostCtx *mev_boost.MEVBoostContext
+		if participantSpec.BuilderNetworkParams != nil {
+			mevBoostLauncher := mev_boost.MEVBoostLauncher{
+				ShouldCheckRelay: true,
+				RelayEndpoints:   participantSpec.BuilderNetworkParams.RelayEndpoints,
+			}
+			mevBoostServiceId := services.ServiceID(fmt.Sprintf("%v%v", mevBoostServiceIdPrefix, idx))
+			mevBoostCtx, err = mevBoostLauncher.Launch(enclaveCtx, mevBoostServiceId, networkParams.NetworkID)
+			if err != nil {
+				return nil, 0, stacktrace.Propagate(
+					err, "could not start mev-boost sidecar",
```

Comment:
I'd recommend something like `Could not start mev-boost sidecar with service ID '%v'` just to give some extra debugging information about explicitly which one failed, but up to you on that one